lib/internal/lambda-events.js

```diff
@@ -83,6 +83,33 @@ function convertScheduledEvent(app, event, context) {
   };
 }
 
+function isAPIGatewayAuthorizerEvent(event) {
+  return !!event.type &&
+    event.type == 'TOKEN' &&
+    !!event.authorizationToken &&
+    !!event.methodArn;
+}
+
+function convertAPIGatewayAuthorizerEvent(app, event, context) {
+  var methodArn = event.methodArn;
+  var splits = methodArn.split(':');
+  var rule = splits[splits.length - 1];
+  var ruleSplits = rule.split('/');
+  var stage = ruleSplits[1];
+  var path = ruleSplits[2];
```

Comment:
Minor nit, but it might be clearer as:
  var path = ruleSplits.slice(2).join('/');